server/src/Dao/AbstractDao.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace App\Dao;
+
+use Psr\Log\LoggerInterface;
+use Psr\Log\NullLogger;
+
+abstract class AbstractDao {
```

Comment:
Why not use a DBAL library like [doctrine/dbal](https://packagist.org/packages/doctrine/dbal) ([docs](http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/))?

Reply:
We could. Let's discuss it offline and if we want we can add it when we work on the endpoint

Reply:
okie dokie, I was just curious. Thought you might have a reason for not using something already out there.